ctx_test.go

```diff
@@ -2119,6 +2119,10 @@ func Test_Ctx_Query(t *testing.T) {
 	utils.AssertEqual(t, "john", c.Query("search"))
 	utils.AssertEqual(t, "20", c.Query("age"))
 	utils.AssertEqual(t, "default", c.Query("unknown", "default"))
+	utils.AssertEqual(t, 0, c.QueryInt("20"))
```

Comment:
please more this to a new test -> don´t mix it

Reply:
Ok Sure.

Reply:
you forgot this

Reply:
Done.